build/notebooks/table_variants_missense.md

```diff
@@ -5409,7 +5672,7 @@ prologue = r"""
 \cmidrule(r){13-14}
 Position\tablefootnote{Position relative to AgamP3 reference sequence, chromosome arm 2L.} & 
 \emph{Ag}\tablefootnote{Codon numbering according to transcript \texttt{AGAP004707-RA} in geneset AgamP4.4.} & 
-\emph{Md}\tablefootnote{Codon numbering according to @@TODO.} & 
+\emph{Md}\tablefootnote{Codon numbering according to \emph{Musca Domesticus Vgsc}.} & 
```

Comment:
Change "Musca Domesticus" to "Musca domestica". Also, is there a citation we can give for this? Either a transcript accession in a public database, or a reference to a paper? What did Dong et al do in their review?